src/ContainerInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Dhii\Data\Container;
+
+use Psr\Container\ContainerInterface as BaseContainerInterface;
+use Dhii\Util\String\StringableInterface as Stringable;
+use Dhii\Data\Container\Exception\NotFoundExceptionInterface;
+use Dhii\Data\Container\Exception\ContainerExceptionInterface;
+
+/**
+ * Represents something that can have data retrieved by key.
+ *
+ * @since [*next-version*]
+ */
+interface ContainerInterface extends
```

Comment:
`get()` implies `has()`. `get()` + `has()` = exactly what `ContainerInterface` is.